modules/git/ref.go

```diff
@@ -84,6 +84,18 @@ func RefNameFromCommit(shortName string) RefName {
 	return RefName(shortName)
 }
 
+func RefNameFromRefTypeAndShortName(refType RefType, shortName string) RefName {
```

Comment:
You should never need it.

Why you know "RefType" but don't know the full "RefName" when using the ref?